package-lock.json

```diff
@@ -5,6 +5,7 @@
   "requires": true,
   "packages": {
     "": {
+      "name": "azure-ad-plugin",
```

Comment:
Do we need this here? Looks like IJ added it when running yarn install 🤔

Reply:
Harmless